core/src/apps/ethereum/layout.py

```diff
@@ -214,9 +214,9 @@ async def should_show_domain(name: bytes, version: bytes) -> bool:
     domain_version = decode_typed_data(version, "string")
 
     para = (
-        (ui.NORMAL, TR.ethereum__name_and_version),
-        (ui.DEMIBOLD, domain_name),
-        (ui.DEMIBOLD, domain_version),
+        (TR.ethereum__name_and_version, False),
```

Comment:
also here: probably you need more parameters for the para array, otherwise what previously was demibold is now normal. is_data selects mono font so perhaps is_title or something